File: Modul 3/PRAK304-2410817110020-AkmalRizki.py
angka = int(input())

if angka == 0:
    print("Nol")
elif angka >= 1 and angka < 10:
    print("Satuan")
elif angka == 10:
    print("Puluhan")
elif angka > 10 and angka < 20:
    print("Belasan")
elif angka > 20 and angka < 100:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Modul 3/PRAK304-2410817110020-AkmalRizki.c
#include <stdio.h>

int main () {
    int angka;
    scanf("%d", &angka);

    if(angka == 0) {
        printf("Nol");
    }
    else if(angka >= 1 && angka < 10) {
        printf("Satuan");
    }
    else if(angka == 10) {
        printf("Puluhan");
    }
    else if(angka > 10 && angka < 20) {
        printf("Belasan");
    }
    else if(angka > 20 && angka < 100) {
        printf("Puluhan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

    return 0;
}